lib/rawHttpRequest/formatOutput.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (body, params, res) => {
+	if (body instanceof Buffer) {
```

Comment:
Do we not want to do anything with files here?

I suppose it would require all connectors to have file support in them, but the download file feature and content disposition support could be useful?

Reply:
Is file handling support something you want to enable for all connectors with Raw HTTP Request then?

I'll create a separate PR for it.